db/seeds.d/75-job_templates.rb

```diff
@@ -3,7 +3,12 @@
     JobTemplate.without_auditing do
       Dir[File.join("#{Katello::Engine.root}/app/views/foreman/job_templates/**/*.erb")].each do |template|
         sync = !Rails.env.test? && Setting[:remote_execution_sync_templates]
-        JobTemplate.import!(File.read(template), :default => true, :locked => true, :update => sync)
+        # import! was renamed to import_raw! around 1.3.1
+        if JobTemplate.respond_to?('import_raw!')
+          JobTemplate.import_raw!(File.read(template), :default => true, :locked => true, :update => sync)
+        else
+          JobTemplate.import!(File.read(template), :default => true, :locked => true, :update => sync)
```

Comment:
sorry I was on PTO...


> Is there a "public interface" method that could be used instead of calling this internal method that can change?

I don't think there's a public interface, mind to open the RFE on REX? I suppose there should be one like we have for registering features. @adamruzicka or @iNecas who work on the plugin might know better about the plan. I hope we can introduce template import in core which would sort of dictate how the import method should look like. That would actually move a lot of code from foreman_templates and REX under the same roof.

> Also, do we still need to support the older method?

Supporting the older method is not mandatory but I suppose is a good practice. Would you prefer forcing users that updates Katello to also update REX at the same time? I'm sure I won't rembember that user must have REX 1.3.1 (not 1.3.0) with Katello 3.4.1.